src/styles/components/_attachments-counter.scss

```diff
@@ -3,7 +3,7 @@
   display: inline-block;
   font-weight: 400;
   text-decoration: none;
-  white-space: nowrap;
+  white-space: nowrap; /* stylelint-disable-line declaration-property-value-disallowed-list */
```

Comment:
Deze nowrap is nodig aangezien een attachments nummer anders onder het icoon komt in tabellen. En aangezien er alleen maar nummers kunnen staan, is dit niet bezwaarlijk